src/applications/edu-benefits/feedback-tool/helpers.js

```diff
@@ -167,3 +178,61 @@ const applicantRelationshipEventMap = {
 export function recordApplicantRelationship({ formData: { onBehalfOf } }) {
   return <UserInteractionRecorder eventRecorder={recordEvent} selectedValue={onBehalfOf} trackingEventMap={applicantRelationshipEventMap}></UserInteractionRecorder>;
 }
+
+/**
+ * A helper that takes data from the SchoolSelectField back end and transforms
+ * it to a valid format as specified by the FEEDBACK-TOOL's
+ * educationDetails.school.address schema.
+ *
+ * @param {*} _ - Object with address fields
+ * @returns {Object} An Object that passes the FEEDBACK-TOOL's educationDetails.
+ * school.address schema.
+ */
+export function transformSearchToolAddress({ address1, address2, address3, city, country, state, zip }) {
+  const isDomesticAddress = country === 'USA';
+  if (isDomesticAddress) {
+    return {
+      country: 'United States',
+      street: typeof address1 === 'string'
```

Comment:
Do we anticipate that these might not be strings? In what situation would that be the case?

Reply:
no. but they might be null or maybe undefined. and we don't want to `slice()` on something we can't slice.

Reply:
That makes sense, if it exists it will be a string, so we can simply could do something like:
```
street: address1 && address.slice(....)
```

But what you've got works great.

Reply:
Ha, yours is totally better. Thanks for catching this!